index.js

```diff
@@ -31,7 +31,7 @@ function getPlugins(appOrAddon) {
 // Regular expression to extract the file extension from a path.
 const EXT_RE = /\.[^\.]+$/;
 
-let fileLookup = null;
+let fileLookup = {};
```

Comment:
Hmm, I actually think that #182 was not the right path forward here. Keeping this in module scope means we don't support running multiple projects within the same process.

Can you tweak things to instead revert the changes in #182 (which will bring back `this.fileLookup`) and then initialize it in `init`? It will have the same basic result as this PR does now (and fix the issue that #182 was also trying to fix), but will avoid that downside...

Reply:
Makes total sense, done. Can you please take another look?